### DISCUSSION TITLE
Modules Setting: data reset button

### DISCUSSION THREAD

**Discussion by satyajittalukder** [getdokan/storegrowth-sales-booster#423] [Feature Request]
## What

The problem you're facing is that the current plugin modules lack a reset button, which makes it difficult for users to revert back to the original state of the data. This lack of functionality leads to a poor user experience.

### Problem

The `absence` of a reset button in each module's setting field/page prevents users from easily reverting back to the initial state of the data that was present when they installed the plugin. Users currently have no way to reset the settings to their original values.

### Solution

To improve the user experience and allow `users` to revert back to the original state of the data, we can add a `reset button` for each `setting field/page`. This `reset button` will `reset `the settings to their `default values`, providing users with a simple way to `restore` the original configuration.

**Comment by imtiaz-pranto** [comment]
Converting it into discussion for further processing
